Source/WebKit/Platform/IPC/Encoder.cpp

```diff
@@ -48,10 +47,10 @@ static inline std::span<uint8_t> allocBuffer(size_t size)
 #if OS(DARWIN)
     auto* buffer = static_cast<uint8_t*>(mmap(0, size, PROT_READ | PROT_WRITE, MAP_ANON | MAP_PRIVATE, -1, 0));
```

Comment:
Probably not worth it in this patch, but eventually we're going to want to make an mmapSpan wrapper.

Reply:
We should make it have an interface similar to `MallocSpan`.